### DISCUSSION TITLE
@peakSet Column Names

### DISCUSSION THREAD

**Discussion by agalianese** [GreenleafLab/ArchR#867] [Questions / Documentation] [answered]
Hi, I was just curious about the meaning for the column names for the proj@peakSet command. What does score, replicateScoreQuantile, and groupScoreQuantile mean and how are they calculated? Additionally, how do distToGeneStart and distToTSS differ? Is this due to an gene allele having a potential different TSS due to a mutation? Thanks!

**Comment by rcorces** [comment, marked as answer]
Apologies for the delay. This wasnt a short answer.

When you create a `peakSet` using `addReproduciblePeakSet()` this runs an iterative overlap procedure where peaks are identified based on the pseudobulk replicates created using `addGroupCoverages()`. Using these psuedobulk replicates, ArchR identifies peaks that are reproducible across those replicates for each group indicated by the `groupBy` argument. The score of those peak calls is stored in the `score` column and the quantile rank of that score for the individual pseudobulk replicate that it was identified in is stored in `replicateScoreQuantile`. After creating these replicate-based peak calls, ArchR then creates a merged/union peak set across the group. The quantile rank of the peak `score` across the group is stored in `groupScoreQuantile` and the group it comes from is stored in `GroupReplicate`.

The difference between `distToGeneStart` and `distToTSS` depend on your `geneAnnotation`.
gene starts are determined by `resize(geneAnnotation$genes, 1, "start")`
TSSs are determined by `resize(geneAnnotation$TSS, 1, "start")`
In the default hg38 `geneAnnotation`, there are 23,274 genes and 49,052 TSSs.

**Reply by agalianese** [reply]
Great! Thank you so much!

**Comment by agalianese** [comment]
Hi, as a follow up, I am trying to find how the overlap between the ArchR Clusters and some peaks of interest. I then want to normalize by the amount of peaks. 

My peakSet has the group replicate as the cluster, then either Rep1 or Rep2 (below is an example for C1). Is there any significant differences between the 2, or just an output of the peak calling? Also, is there another way to determine the amount of peaks per cluster, or just this? Thank you so much!


```
C1._.Rep1 | 852 |   
C1._.Rep2 | 1974
```

**Reply by rcorces** [reply]
In the future, it helps other users more if you open a new discussion topic so that things are easier to search and find.

For what its worth, I dont understand your question enough to provide a clear answer. I think its important to remember that peak calling is highly imperfect, highly dependent on total sequencing depth (or cells per cluster in scATAC), and doing raw overlaps or looking at the number of peak calls from a given sample is somewhat dubious.

**Comment by agalianese** [comment]
Ok, that's what I thought. Thanks!
